src/checkbox_todo.js

```diff
@@ -0,0 +1,16 @@
+export function toggleChkBox(todos, index) {
+  todos[index].completed = !todos[index].completed;
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
+export const clearCompletedChkBox = (todos) => {
+  todos.forEach((todo, index) => {
+    if (todo.completed) {
+      todos.splice(index, 1);
+    }
+  });
```

Comment:
- Good job done so far, according to the project requirements, when the `clear all button` is clicked, it should delete all completed todos. But this isn't so in your project as it deletes all to-dos except one. Please see the image below. I suggest using the `Array.filter` method to filter all completed todos 👍 👍 

Your project(after clicking on the completed todos):
![wwrgg](https://user-images.githubusercontent.com/73607512/236632218-67973fa3-68ba-46e4-83c1-85006a347d27.PNG)

Your project(When I click on clear all button):
![rrgwr](https://user-images.githubusercontent.com/73607512/236632254-b253189e-6387-4d21-8c36-09bbf3e5a4a0.PNG)